mockwebserver/src/test/java/mockwebserver3/MockWebServerTest.java

```diff
@@ -456,7 +456,7 @@ public void tearDown() throws Exception {
 
     HttpUrl requestUrl = request.getRequestUrl();
     assertThat(requestUrl.scheme()).isEqualTo("http");
-    assertThat(requestUrl.host()).isEqualTo(server.getHostName());
+    assertThat(requestUrl.host()).isEqualToIgnoringCase(server.getHostName());
```

Comment:
If it's gonna return lowercase anyway we should remove the case insensitive asserts.